memory-viz/src/types.ts

```diff
@@ -0,0 +1,25 @@
+interface DrawnObject {
```

Comment:
Use the `export` keyword for these types, and then import them explicitly in `style.ts`.

Reply:
Also rename this type to `DrawnEntity`, since the word "object" has another meaning in the context of memory model diagrams.